docs/content/integrations/dbt.mdx

```diff
@@ -1,380 +1,159 @@
-            )
+@repository
+def my_repo():
+    return [
+        dbt_assets,
+        ScheduleDefinition(
+            job=run_something,
+            cron_schedule="@daily",
+        ),
+        ScheduleDefinition(
+            job=run_everything,
+            cron_schedule="@weekly",
         )
-    return dbt_result
+    ]
 ```
 
-**dbt CLI: Set the dbt profile and target to load**
+See (docs for running stuff on a schedule) to learn more.
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_config_profile_and_target endbefore=end_marker_dbt_cli_config_profile_and_target dedent=4
-from dagster_dbt import dbt_cli_resource
-
-from dagster import job
+## Defining Downstream Dependencies
 
-config = {"profile": PROFILE_NAME, "target": TARGET_NAME}
+Dagster allows you to define assets that are downstream of specific dbt models. One property of dbt based assets is that the external tool (in this case, dbt), handles the loading/storing of those models internally, rather than Dagster directly storing/loading the tables that are updated.
 
-@job(resource_defs={"dbt": dbt_cli_resource.configured(config)})
-def my_job():
-    # ...
-```
+This means that there's a range of ways that we might want to load a dbt model as input to a python function. For example, you might want to load the contents of the table as a Pandas dataframe, or you might want to load the contents into a PySpark session.
 
-**dbt CLI: Set the path to the dbt executable**
+To define exactly how you want models to be loaded, you can specify an input manager on the downstream asset.
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_config_executable endbefore=end_marker_dbt_cli_config_executable dedent=4
-from dagster_dbt import dbt_cli_resource
+```python
+from dagster import asset
 
-from dagster import job
-
-config = {"dbt_executable": "path/to/dbt/executable"}
-
-@job(resource_defs={"dbt": dbt_cli_resource.configured(config)})
-def my_job():
-    # ...
+@asset(
+    ins={"model_name": AssetIn(input_manager_key="pandas_dbt_input_manager")}
+)
+def my_downstream_asset(model_name):
+    return
 ```
 
-**dbt CLI: Select specific models to run**
+## Customizing AssetKeys
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_config_select_models endbefore=end_marker_dbt_cli_config_select_models dedent=4
-from dagster_dbt import dbt_cli_resource
+Dagster automatically generates AssetKeys for each dbt model in the project, as well as any of the sources used by these models.
 
-from dagster import job
+The default asset key for a model will be the model name, and the default name for a source in the project will be the name of the source alongside the name of the source table.
 
-config = {"models": ["my_dbt_model+", "path.to.models", "tag:nightly"]}
+For example, a model defined in `my_model.sql` will be given the asset key `my_model`, while the source table defined in the following `sources.yaml` will be `jaffle_shop/orders`.
```

Comment:
```suggestion
For example, a model defined in `my_model.sql` will be given the asset key `my_model`, while the source table defined in the following `sources.yaml` will be `jaffle_shop/orders`:
```

Replaced the period with a colon to connect the copy to the example that follows.